src/locale/languages/tr.js

```diff
@@ -0,0 +1,318 @@
+        'Sohbet ederek iletişim kurun, pull request\'leri merge edin ve daha fazlası',
+      signInButton: 'GİRİŞ YAP',
+    },
+    welcome: {
+      welcomeTitle: 'GitPoint\'e Hoşgeldiniz',
+    },
+    events: {
+      welcomeMessage:
+        'Hoşgeldiniz! Bu, haber kaynağınızdır - izlediğiniz repository\'lerde ve takip ettiğiniz kişilerle ilgili son etkinlikleri takip etmenize yardımcı olur.',
+      commitCommentEvent: 'commit\'e yorum yapıldı',
+      createEvent: '{{object}} oluşturuldu',
+      deleteEvent: '{{object}} silindi',
+      issueCommentEvent: '{{type}} {{action}}',
+      issueEditedEvent: '{{type}}\'daki yorum {{action}}',
+      issueRemovedEvent: '{{type}}\'den yorum {{action}}',
+      issuesEvent: '{{action}} issue',
+      publicEvent: {
+        action: 'made',
+        connector: 'public',
+      },
+      pullRequestEvent: 'pull request {{action}}',
+      pullRequestReviewEvent: 'pull request incelemesi {{action}}',
+      pullRequestReviewCommentEvent: 'pull request\'e {{action}}',
+      pullRequestReviewEditedEvent: 'pull request\'deki yorum {{action}}',
+      pullRequestReviewDeletedEvent: 'pull request\'deki yorum {{action}}',
+      releaseEvent: 'release {{action}}',
+      atConnector: 'de',
+      toConnector: 'dan',
+      types: {
+        pullRequest: 'pull request',
+        issue: 'issue',
+      },
+      objects: {
+        repository: 'repository',
+        branch: 'dal',
+        tag: 'etiket',
+      },
+      actions: {
+        added: 'eklendi',
+        created: 'oluşturuldu',
+        edited: 'düzenlendi',
+        deleted: 'silindi',
+        assigned: 'atandı',
+        unassigned: 'atanmadı',
+        labeled: 'etiketli',
+        unlabeled: 'etiketsiz',
+        opened: 'açıldı',
+        milestoned: 'milestoned',
+        demilestoned: 'demilestoned',
+        closed: 'kapandı',
+        reopened: 'yeniden açıldı',
+        review_requested: 'inceleme istendi',
+        review_request_removed: 'inceleme isteği silindi',
+        submitted: 'gönderildi',
+        dismissed: 'reddedildi',
+        published: 'yayımlandı',
+        publicized: 'Halka açıldı',
+        privatized: 'Özel yapıldı',
+        starred: 'yıldızlı',
+        pushedTo: '\'a push edildi',
+        forked: 'fork edildi',
+        commented: 'yorum yaptı',
+        removed: 'silindi',
+      },
+    },
+    profile: {
+      orgsRequestApprovalTop: 'Tüm organizasyonlarınızı göremiyor musunuz?',
+      orgsRequestApprovalBottom: 'Onlar için onay istemeniz gerekebilir',
+      codePushCheck: 'Güncellemeleri kontrol et',
+      codePushChecking: 'Güncellemeler kontrol ediliyor...',
+      codePushUpdated: 'Uygulama güncel',
+      codePushAvailable: 'Güncelleme mevcut!',
+      codePushNotApplicable: 'Hata ayıklama modunda uygulanamaz',
+    },
+    userOptions: {
+      title: 'Ayarlar',
+      language: 'Dil',
+      privacyPolicy: 'Gizlilik Politikası',
+      signOut: 'Çıkış Yap',
+    },
+    privacyPolicy: {
```

Comment:
@abdurrahmanekr can you translate anything from the policy policy block? It would be great to translate everything, although it is difficult, because a lot of text.